packages/rocketchat-livechat/server/publications/livechatMonitoring.js

```diff
@@ -0,0 +1,32 @@
+import moment from 'moment';
+
+Meteor.publish('livechat:monitoring', function() {
+	if (!this.userId) {
+		return this.error(new Meteor.Error('error-not-authorized', 'Not authorized', { publish: 'livechat:monitoring' }));
+	}
+
+	const date = {
```

Comment:
I don't agree with the use of `moment` here.

Reply:
If not, we will be subscribing to all the live chat data (massive), but what we need is only that day's data.

Reply:
I don't agree with using `moments` on server-side, it's not related in passing the date params.. 
You can pass the date by params without using `moments` on server side, that is what I meant.